scripts/world_bank/health_nutrition_population/population_get_mcf.py

```diff
@@ -0,0 +1,35 @@
+def get_mcf(series_lst):
+    '''
+    gets mcf by splitting description and using fstrings
+    '''
+    nodes = []
+    #don't  use df; automate
+    with open("output\World_bank_hnp_population.mcf", 'w', encoding = 'utf-8') as file:
+        print(end = '')
+        mcf = ''
+        for current_series in series_lst:
+            print('\n'+current_series)
+            statvars = ['Node', 'typeOf', 'description', 'populationType',
```

Comment:
these are technically properties of the statvars, not statvars themselves.
```suggestion
            properties = ['Node', 'typeOf', 'description', 'populationType',
```

Reply:
Corrected this